### PR TITLE
WRP-9591: Fixed console errors in console app

### PR DESCRIPTION
**Pull request by ion-andrusciac-lgp**
### Checklist 

* [X] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [ ] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [X] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
Checked the app and fixed triggered console errors.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Unrecognized tag errors were fixed by the following examples:
https://github.com/facebook/react/issues/11184#issuecomment-335942439
https://stackoverflow.com/questions/48751919/how-can-i-suppress-the-tag-some-tag-is-unrecognized-in-this-browser-warning

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)

### Links
[//]: # (Related issues, references)
WRP-9591

### Comments

Enact-DCO-1.0-Signed-off-by: Ion Andrusciac ion.andrusciac@lgepartner.com

### CODE REVIEW

File: console/src/views/ThemeSettings.js [hunk 1]
```diff
@@ -122,7 +122,7 @@ const ThemeSettingsBase = kind({
 		}
 	},
 
-	render: ({css, onChange, onSelect, onSendSkinSettings, prevIndex, ...rest}) => (
+	render: ({css, onChange, onSelect, prevIndex, ...rest}) => (
```
Comment (daniel-stoian-lgp) [inline console/src/views/ThemeSettings.js]: the error did not dissapear. please add a
`delete rest.onSendSkinSettings;`

File: console/src/App/AppContextProvider.js [hunk 2]
```diff
@@ -186,7 +186,8 @@ class AppContextProvider extends Component {
 	};
 
 	loadUserSettings = (userId) => {
-		return JSON.parse((global.localStorage && global.localStorage.getItem(`user${userId}`)) || '{}') || this.getDefaultUserSettings(userId);
+		const user = JSON.parse((global.localStorage && global.localStorage.getItem(`user${userId}`)) || '{}');
+		return Object.keys(user).length !== 0 ? user : this.getDefaultUserSettings(userId);
```
Comment (daniel-stoian-lgp) [inline console/src/App/AppContextProvider.js:190]: can you give me a test scenario in order to catch the error that was solved? i couldn't find it myself

**Review by daniel-stoian-lgp — COMMENTED** [review]
only 2 comments

File: console/src/App/AppContextProvider.js (hunk 2 above)
Comment (ion-andrusciac-lgp) [reply console/src/App/AppContextProvider.js:190]: on "reset current user" or "restart demo", this method return always an empty object

**Review by daniel-stoian-lgp — APPROVED** [review]

**Review by stanca-pop-lgp — APPROVED** [review]

**Pull request merged**